### ISSUE TITLE
No results displayed when filter by Brodie + Metabolomics

### ISSUE THREAD

**Issue by pvangay** [microbiomedata/nmdc-server#333]
Selecting PI = Brodie and Omic type = MetaB results in the following error, with no data displayed in the results list even though the study lists there being 52 MetaB data.
![image](https://user-images.githubusercontent.com/7374085/110719586-effdf000-81c1-11eb-90aa-7accd6db37b7.png)

**Comment by jbeezley** [comment]
This has to do with these biosamples which are still missing:
```
igsn:IEWFS000K
igsn:IEWFS000B
igsn:IEWFS000I
igsn:IEWFS000A
igsn:IEWFS000J
```
In mongo, they are missing the `_study_id` attribute I'm relying on in ingest.  @dwinston I believe you added these biosamples manually, would you be able to add the `_study_id` attribute as well?

**Comment by dwinston** [comment]
@jbeezley Done. Re-ordered my pipeline to place the `_study_id`-ensuring step after the additional-insertions step.

**Comment by dehays** [comment]
Closing this one.  Either this was another issue which was resolved in March with Donny setting the _study_id.  Or, this is a different issue than one related to the samples mentioned above and is actually the same issue as microbiomedata/nmdc-metadata#341 

I think it is the latter looking at the 52 Brodie - Metabolomics omics_processing entities - which have no input samples at all. (The portal is showing the associated samples - and that set is the empty set.  I think I commented in the other issue that omics_processing (instrument processes) without sample inputs should be invalid.  So schema change and then metadata bug.
